glutton.go

```diff
@@ -221,6 +221,7 @@ func (g *Glutton) registerHandlers() {
 				}
 
 				done := make(chan struct{})
+				g.logger.Info("Connection is being closed.")
```

Comment:
The connection is not actually closed here. Also this should be a debug message only and should have some information about the connection (maybe hosts and ports involved)